test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelperSampleTest.cs

```diff
@@ -32,9 +34,10 @@ public class TagHelperSampleTest
         };
 
         // Path relative to Mvc\\test\Microsoft.AspNet.Mvc.FunctionalTests
+        private readonly ILoggerFactory _loggerFactory = new TestLoggerFactory();
```

Comment:
comment above should be moved below to sit above the `_services` variable like before.